README.md

```diff
@@ -54,7 +54,8 @@ The `${HOMEBREW_PREFIX}/opt/socket_vmnet` directory is usually symlinked to `../
 
 Run the following command to start the daemon:
 ```bash
-mkdir -p ${HOMEBREW_PREFIX}/var/run && sudo ${HOMEBREW_PREFIX}/opt/socket_vmnet/bin/socket_vmnet --vmnet-gateway=192.168.105.1 ${HOMEBREW_PREFIX}/var/run/socket_vmnet
+mkdir -p ${HOMEBREW_PREFIX}/var/run && \
```

Comment:
Just two lines
```suggestion
sudo mkdir -p ${HOMEBREW_PREFIX}/var/run
sudo ${HOMEBREW_PREFIX}/opt/socket_vmnet/bin/socket_vmnet --vmnet-gateway=192.168.105.1 ${HOMEBREW_PREFIX}/var/run/socket_vmnet
```

Reply:
Also please squash commits

Reply:
@AkihiroSuda before we moving on to adding `sudo` to `mkdir` did you test if this is necessary on intel macs? https://github.com/lima-vm/socket_vmnet/pull/28#discussion_r1102151235

Reply:
Sorry, sudo is not needed (when you are in the `admin` group)

Reply:
thanks for checking 👍🏼
just reflected everything and the commits are squashed as well